mobify/platform-scaffold
# Update package.json to beta 3

Updates the native folder to the latest released version of mobify-progressive-app-sdk

## Changes
- Update to beta 3
- Update astro-client

## How to test-drive this PR
- `cd native`
- `npm run deps`
- `cd ../web`
- Run `npm run dev`
- Open the `scaffold.xcworkspace` file in the native/ios folder
- Make sure you have the `scaffold` target selected.
- Build and run the app (`CMD+R`)
